Dockerfile.skupper-cli

```diff
@@ -23,5 +23,5 @@ RUN useradd --uid 10000 runner
 USER 10000
 
 WORKDIR /app
-COPY --from=builder /go/src/app/bootstrap .
-CMD ["/app/bootstrap"]
+COPY --from=builder /go/src/app/skupper .
+CMD ["/app/skupper"]
```

Comment:
FWIW, I think we should consider moving to setting skupper as the ENTRYPOINT instead of the CMD.

As I understand it the only purpose of this container image is to run CLI, so I think setting it as the entrypoint (rather than the default /bin/bash) makes sense.

Example Usage with CMD:
`podman run -it quay.io/skupper/skupper-cli:v2-latest` -> prints the CLI help :heavy_check_mark: 
`podman run -it quay.io/skupper/skupper-cli:v2-latest system --help` -> `Error: crun: executable file `system` not found in $PATH` :x: 

Now I'm confused, and end up running `podman run -it quay.io/skupper/skupper-cli:v2-latest skupper`, It's not on the path so that doesn' work. Eventually I run`podman run -it quay.io/skupper/skupper-cli:v2-latest /bin/sh` to manually search for the skupper CLI or `podman image inspect` to sort out what that default CMD was.

With an ENTRYPOINT, the UX is smoother and there is no need to understand the internals of the image.

Reply:
Also, while on image nits.

I had thought I recalled we had decided that we wanted to not repeat the namespace in image names - i.e. quay.io/skupper/router instead of quay.io/skupper/skupper-router. So I'd assume that'd apply here as well: quay.io/skupper/cli.

Reply:
You are right, done! (6b7f1a2ab06b2d70cd54251dd20a8eb24dea358d)